frontend/src/assets/img/close.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you confer with @MelissaBraxton about this? I'm not sure a "close" is part of the standards. Could this just be the text "close"

Reply:
@lauraGgit this was copied from uswds https://standards.usa.gov/assets/img/close.svg